README.md

```diff
@@ -177,3 +189,23 @@ improvements. See [Contributing.md](Contributing.md) for more details.
 See [https://spglib.readthedocs.io/](https://spglib.readthedocs.io/) for further
 documentation. See the [doc documentation](doc/README.md) for more information on
 how to contribute to the documentation.
+
+[ci-badge]: https://github.com/LecrisUT/CMake-Template/actions/workflows/ci.yaml/badge.svg?branch=main&event=push
```

Comment:
Is this link correct? It seems to link to your action.

Reply:
Oops, thanks for the catch :D